draft-packages/modal/KaizenDraft/Modal/Presets/InputEditModal.scss

```diff
@@ -20,5 +20,4 @@
   background: $kz-var-color-stone;
   padding: $ca-grid $ca-grid $ca-grid * 1.5;
   border-bottom: 1px solid $ca-border-color;
```

Comment:
This one looks like it was on me 😬 

I believe the rationale in [this PR](https://github.com/cultureamp/kaizen-design-system/pull/530/files) was to ensure no matter what a consumer put in the body (i.e. they forget to use the Heading / Paragraph component) it always rendered Inter typography.

Reply:
Ah I see. I wonder if a css reset or base styles would be a better way of ensuring that - I believe you have a PR for that in the works. Otherwise it seems like we'd have to do that in every component that accepts children. I don't know if it's possible to force people via Typescript to not just use plain text nodes, or that would be another approach.

Reply:
I _think_ this is safe enough to merge.

Reply:
Yeah, I agree. It's low risk and low impact. The behaviour of engineers using Kaizen seems to be to reach towards Heading/Paragraph (or Text >.>) when needing copy. This is definitely something the shell component should take care off too 👍🏻

Reply:
Yeah, I think people are pretty accustomed to using Heading/Paragraph now which is great.